libindy/src/utils/crypto/sign/sodium.rs

```diff
@@ -0,0 +1,118 @@
+extern crate sodiumoxide;
```

Comment:
This module should be called ed25519, but not sign. It is better to include other ed25519 related operations to this module.